README.md

```diff
@@ -40,6 +40,17 @@ glslang is a GLSL and HLSL to SPIR-V compiler, usable both as a command-line too
 * Contact: David Neto <dneto@google.com>
 * Compiler requirements: C++11 compatible compiler (MSVC 2013+ on Windows)
 
+## DXC
+DirectXShaderCompiler (DXC) is Microsoft's next-gen official HLSL compiler, based on LLVM/Clang.
+It can also compile HLSL into SPIR-V, contributed by Google.
+
+* Repository: https://github.com/Microsoft/DirectXShaderCompiler
+* License: University of Illinois Open Source License
+* Contact: Lei Zhang <antiagainst@google.com>
+* Platform: Windows
+* Compiler requirement: Visual Studio 2017
```

Comment:
The wiki also seems to state support for Visual Studio 2015 Update 3.

Reply:
IIRC, we used to have problems of compiling it using VS 2015. VS 2015 path is not tested, at least publicly.